themes/ddbasic/sass/components/ting-object/material-item.scss

```diff
@@ -189,7 +189,6 @@
       }
 
       > div {
-        margin-right: 30px;
```

Comment:
I wonder why the changes in the PR require the removal of this styling. It does not seem to be related to the change at hand.

Reply:
I really don't remember. There where some issues with som margin in the payment site which lead back to this. But every time i try to figure out why and which commit ... github JS stops working here in Chrome